src/components/LandingText/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Typography from '@/components/Typography';
+import Link from 'next/link';
+import styles from './style.module.scss';
+
+const LandingText = () => {
+  return (
+    <div className={styles.container}>
+      <Typography variant="title/small" className={styles.subheading}>
+        April 5-6, 2025 • hosted by <span className={styles.gradient}>ACM at UC San Diego</span>
```

Comment:
typographical nitpick but you could use an n dash here

```suggestion
        April 5–6, 2025 • hosted by <span className={styles.gradient}>ACM at UC San Diego</span>
```